web/client/api/geofence/__tests__/RuleService-test.js

```diff
@@ -40,7 +40,7 @@ describe('RuleService API for GeoFence StandAlone', () => {
     it('getRulesCount', (done) => {
         const PARAMS = { roleName: "ADMIN" };
         mockAxios.onGet().reply(config => {
-            expect(config.url).toBe(`${BASE_URL}/rules/count`);
```

Comment:
these changes are related to new mock adapter?

Reply:
From changelog: 
--> this fix Do not modify config.url when using a relative baseURL (resolves [#1628](https://github.com/axios/axios/issues/1098)) (https://github.com/axios/axios/pull/1098)

now config.url is not modified by axios adding the baseURL anymore.

Reply:
thanks for the clarification